src/routes/v2/authenticated/review.ts

```diff
@@ -634,6 +635,52 @@ export class ReviewsRouter {
     return res.status(200).send()
   }
 
+  getComments: RequestHandler<
+    { siteName: string; requestId: number },
+    CommentItem[] | ResponseErrorBody,
+    never,
+    unknown,
+    { userWithSiteSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { siteName, requestId } = req.params
+    const { userWithSiteSessionData } = res.locals
+    // Step 1: Check that the site exists
+    const site = await this.sitesService.getBySiteName(siteName)
+    if (!site) {
+      return res.status(404).send({
+        message: "Please ensure that the site exists!",
+      })
+    }
+
+    // Step 2: Retrieve comments
+    const comments = await this.reviewRequestService.getComments(
+      userWithSiteSessionData,
+      site,
+      requestId
+    )
+
+    return res.status(200).json(comments)
+  }
+
+  createComment: RequestHandler<
+    { siteName: string; requestId: number },
+    string | ResponseErrorBody,
```

Comment:
doesn't seem to return `ResponseErrorBody` .-.

Reply:
Removed in https://github.com/isomerpages/isomercms-backend/pull/534/commits/8f8bbb7e163fe141a0d84ec20d433dfb5c99e0a1

Reply:
think you removed the wrong one HAHA

Reply:
oops, fixed in https://github.com/isomerpages/isomercms-backend/pull/534/commits/d60c399848f15ddc5465da95e048103f03902053